packages/mc-scripts/package.json

```diff
@@ -42,6 +54,7 @@
     "autoprefixer": "^10.4.4",
     "babel-loader": "8.2.5",
     "browserslist": "^4.20.2",
+    "cac": "6.7.12",
```

Comment:
New CLI library as a replacement for `mri`, as it has some nicer features.

https://www.npmjs.com/package/cac